### DISCUSSION TITLE
Touch not working on rotated screen

### DISCUSSION THREAD

**Discussion by tarscher** [Joshua-Riek/ubuntu-rockchip#242] [Q&A]
I have developed a webkiosk-like application that utilizes a rotated screen of 90 degrees in Ubuntu. While I can successfully rotate the screen in the Ubuntu user interface, I'm encountering an issue with the touch screen functionality. It appears that the touch input is not properly rotated to align with the screen orientation.

When I disable Wayland it seems that hardware acceleration in Chromium is lacking (which i m in need of)

Any tips on how to solve this

Thanks

**Comment by uxandy** [comment]
Hi there,

I am having the same issue. I think it has to do with the way XWayland is configured. Xinput does not do anything and just returns that it's running XWayland server.

I am on the verge of returning the Orange Pi and going with something else because of this. Does anyone here have any idea if this issue is resolvable with this distro? Thanks.
